quay-enterprise/geo-replication.md

```diff
@@ -0,0 +1,36 @@
+# Georeplication of storage in Quay Enterprise
+
+Georeplication allows for a single globally-distributed Quay Enterprise to serve container images from localized storage.
+
+When setup and enabled, all pushes performed will first save their image data to the preferred storage engine for the Quay Enterprise instance. Following push, the image data will be replicated in the background to other storage engines (configurable). Pulling of images will always make use of the closest *avaliable* storage engine, ensuring availability and performance.
+
+**NOTE: Ensure that there is a high availability storage engine (S3, GCS, RADOS, Swift) in each geographic region and that each region can access each storage engine. Local disk storage is not compatible with georeplication at this time.**
+
+## Visit the Management Panel
+
+Sign in to a super user account and visit `http://yourregister/superuser` to view the management panel:
+
+<img src="img/superuser.png" class="img-center" alt="Enterprise Registry Management Panel"/>
+
+## Enable Storage Replication
+
+1. Click the configuration tab (<span class="fa fa-gear"></span>) and scroll down to the section entitled <strong>Registry Storage</strong>.
+2. Click "Enable Storage Replication".
+3. Add each of the storage engines to which data will be replicated. All storages to be used must be listed and **all regions must have access to _all_ storage engines**.
```

Comment:
This seems quite important but I'm not sure if this sentence makes me 100% what I need to do. So my europe install needs to be able to access Swift in the US?